config/default.vcl

```diff
@@ -0,0 +1,22 @@
+# specify the VCL syntax version to use
+vcl 4.1;
+
+# import vmod_dynamic for better backend name resolution
+import dynamic;
+
+# we won't use any static backend, but Varnish still need a default one
+backend default none;
+
+# set up a dynamic director
+# for more info, see https://github.com/nigoroll/libvmod-dynamic/blob/master/src/vmod_dynamic.vcc
+sub vcl_init {
+        new d = dynamic.director(port = "8013");
+}
+
+sub vcl_recv {
+	# force the host header to match the backend (not all backends need it,
+	# but example.com does)
+	set req.http.host = "nginx";
```

Comment:
@jkachel I think this might have added a side effect on local environment, I was testing the payment while working on [another PR](https://github.com/mitodl/ol-django/pull/96) and the callback after the payment takes me to something like `nginx/callback/result` (can't remember exactly) and this URL wasn't resolvable for obvious reasons.

I'm not sure if I missed anything but I rebuilt containers `docker-compose build` and I could see `varnish` one too when it happend. Or maybe i'm missing something else.

Reply:
Noticed this too.. working on a fix for it.